src/pages/signin/SignInPage.js

```diff
@@ -150,7 +150,7 @@ class SignInPage extends Component {
         return (
             // There is an issue SafeAreaView on Android where wrong insets flicker on app start.
             // Can be removed once https://github.com/th3rdwave/react-native-safe-area-context/issues/364 is resolved.
-            <View style={[styles.signInPage, StyleUtils.getSafeAreaPadding(this.props.insets, 1)]}>
+            <View style={[styles.signInPage, StyleUtils.getSafeAreaPadding({...this.props.insets, bottom: 0}, 1)]}>
```

Comment:
I'm adding in a comment for clarity, but the design asked that the `SignInPage` svg (snakes and hands) to pass under the Home Indicator. This wasn't an issue before the Apple sign in button, but on smaller screen the apple sign in button pushed the svg so that it was cut off by the safe area inset instead of flowing under the Home Indicator.